From 8a3bfebfbcb7c1dab3bec763f07c731372c17ce4 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 25 Jan 2019 15:11:50 +0000
Subject: [PATCH] setup-alpine: setup ntp before repos

we need time to be correct for https certificates when setting up
apkrepos, so we call setup-ntp before setup-apkrepos.

ref #9911
---
 setup-alpine.in | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/setup-alpine.in b/setup-alpine.in
index 9bae702..80e10ff 100644
--- a/setup-alpine.in
+++ b/setup-alpine.in
@@ -186,6 +186,10 @@ if [ -r "$ROOT/etc/profile" ]; then
 	. "$ROOT/etc/profile"
 fi
 
+if ! is_qemu && [ "$rc_sys" != "LXC" ] && [ "$quick" != 1 ]; then
+	$PREFIX/sbin/setup-ntp ${NTPOPTS}
+fi
+
 $PREFIX/sbin/setup-apkrepos ${APKREPOSOPTS}
 
 # lets stop here if in "quick mode"
@@ -195,10 +199,6 @@ fi
 
 $PREFIX/sbin/setup-sshd ${SSHDOPTS}
 
-if ! is_qemu && [ "$rc_sys" != "LXC" ]; then
-	$PREFIX/sbin/setup-ntp ${NTPOPTS}
-fi
-
 if is_xen_dom0; then
 	setup-xen-dom0
 fi
-- 
2.20.1

